security/security_policy_compliance/timemachine.md

```diff
@@ -4,19 +4,22 @@ backup was taken. For this reason, encrypting these backups is important.
 
 ## First-Time Setup
 
-If you are not currently backing up the machine, follow the [Apple provided instructions](https://support.apple.com/en-us/HT201250) for setting up encrypted Time Machine backups to an external disk.
+System backups are not required, but are encouraged for he purpose of business
+continuity. If you choose not to maintain regulare system backups, it is
+_highly_ encouraged that you be able to bring your machine to a workable
+state quickly. For developers, managing your `dotfiles` is encouraged. Setup
```

Comment:
Nitpick: is there a reason why "dotfiles" is encapsulated in backticks? I'm cool with it staying, but I just haven't seen this used outside of a way to quote code or specific file names.

Reply:
Resolved in b218534aaeb36941eeef26f2c86809bc8d07e246.